Movie.h

```diff
@@ -0,0 +1,65 @@
+#include "Genre.h"
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+using namespace std;
+class Movie
+{
+private:
```

Comment:
The class members should be wrapped by 4 spaces(or 1 tab).

Reply:
Changed